src/services/errors.js

```diff
@@ -34,5 +34,9 @@ module.exports= {
 	UserAlreadyExistsError : {
 		name: 'UserAlreadyExistsError',
 		message: 'User already exists'
+	},
+	GifDoesNotExistError : { 
```

Comment:
@beblicarl not seeing where we're throwing this error yet, and this is a creation endpoint issue so we don't seem to have a reason to throw it